lib/profile/sections/section.dart

```diff
@@ -0,0 +1,104 @@
+import 'package:google_fonts/google_fonts.dart';
+import 'package:flutter/material.dart';
```

Comment:
flutter package should appear before google_fonts package, imports should be alphabetical

Reply:
Ooo ok ok!